django/db/backends/postgresql/introspection.py

```diff
@@ -62,7 +63,8 @@ def get_table_list(self, cursor):
                     WHEN c.relispartition THEN 'p'
                     WHEN c.relkind IN ('m', 'v') THEN 'v'
                     ELSE 't'
-                END
+                END,
+                obj_description(c.oid)
```

Comment:
While looking at CockroachDB support, the following from [the PostgreSQL documentation](https://www.postgresql.org/docs/15/functions-info.html#FUNCTIONS-INFO-COMMENT-TABLE) was brought to my attention:
```
obj_description ( object oid ) → text

Returns the comment for a database object specified by its OID alone.
This is deprecated since there is no guarantee that OIDs are unique
across different system catalogs; therefore, the wrong comment might
be returned.
```

Reply:
Good catch :dart: We should add a catalog name.